packages/ringcentral-widgets/components/CallsListPanel/index.js

```diff
@@ -366,14 +393,55 @@ export default class CallsListPanel extends Component {
       />
     );
 
-
-    const historyCall = showSpinner ?
-      <SpinnerOverlay /> :
-      (
-        <div className={classnames(styles.list, className)}>
-          <div className={styles.listTitle}>
-            {onlyHistory ? null : i18n.getString('historyCalls', currentLocale)}
-          </div>
+    const historyCall = showSpinner ? (
+      <SpinnerOverlay />
+    ) : (
+      <div
+        className={classnames(styles.list, className)}
+        ref={this._callListWrapper}
+      >
+        <div className={styles.listTitle}>
+          {onlyHistory ? null : i18n.getString('historyCalls', currentLocale)}
+        </div>
+        {useNewList ? (
```

Comment:
I'd recommend using a
```javascript
const callList= useNewList ? (...) : (...);
```
block before this to make this markup section more readable, or even put it into a renderCallList function if that makes sense.

Reply:
done